client/src/templates/Challenges/redux/create-question-epic.test.js

```diff
@@ -27,4 +28,62 @@ describe('create-question-epic', () => {
       });
     });
   });
+  describe('insertEditableRegions', () => {
+    const multiCertChallengeFiles = [
+      {
+        contents: '<h1>Hello World</h1>',
+        editableRegionBoundaries: [],
```

Comment:
```suggestion
```
The original error was caused by this being absent, so it makes sense to test for that.